### ISSUE TITLE
How is the proposition in session 7 task 2 even valid?

### ISSUE THREAD

**Issue by wunderboyy** [nomeata/incredible#139]
![image](https://github.com/nomeata/incredible/assets/97628152/9a4fda71-2e67-4cf5-ba60-f9d62098d4df)

How is this proposition valid? The premise says that for all x, if r(x) is false then r(f(x)). So r(x) can clearly be false for all x. BUT the conclusion states that there exists an x such that r(x) is true??

**Comment by NicoleRauch** [comment]
> So r(x) can clearly be false for all x.

No, it can’t, because if it is false for a given x, then the premise states that it must be true for f(x). So there must be at least one value for which r is true. Which is the first half of the conclusion 😊

**Comment by nomeata** [comment]
Good explanation! It's worth adding that in our logic, the universe is never empty, (`exists x. True` is an axiom, so to say), else the above would not be a theorem.

**Comment by NicoleRauch** [comment]
There is always something I am missing out on 😆
